deepinv/loss/metric/metric.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import annotations
+from types import ModuleType
+from typing import Optional, TYPE_CHECKING
+
+from torch import Tensor
+from torch.nn import Module
+
+from deepinv.loss.loss import Loss
+from deepinv.loss.metric.functional import complex_abs, norm
+from deepinv.utils.plotting import rescale_img
+
+if TYPE_CHECKING:
+    from deepinv.physics.forward import Physics
+
+
+def import_pyiqa() -> ModuleType:
+    try:
+        import pyiqa
+
+        return pyiqa
+    except ImportError:
+        raise ImportError(
+            "Metric not available. Please install the pyiqa package with `pip install pyiqa`."
+        )
+
+
+class Metric(Loss):
```

Comment:
Do we want to inherit from Loss?

I am afraid that we end up with some circular patterns, eg EILoss using a metric

Reply:
Metric now no longer inherits from loss